<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFileDialog</class>
 <widget class="QDialog" name="NewFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New File</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 63, 65);
color: rgb(230, 230, 230)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>File Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>File Type:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="fileName">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 2px solid rgb(230, 230, 230);
    border-radius: 1px;
}
QLineEdit:hover {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}

QLineEdit:focus {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fileTypeComboBox">
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border: 2px solid rgb(230, 230, 230);
    border-radius: 1px;
}
QComboBox:hover {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}

QComboBox:focus {
	background-color: rgb(64, 64, 64);
	border: 2px solid #8f8f91;
	border-color: rgb(60, 187, 255);
}
</string>
           </property>
           <item>
            <property name="text">
             <string>Program ( Python File )</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}
</string>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 3px;
	height: 30px;
	width: 60;
}
QPushButton:hover {
	background-color: rgb(60, 187, 255);
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
